src/Durable/StartDurableTimerCommand.cs

```diff
@@ -22,15 +22,15 @@ public class StartDurableTimerCommand : PSCmdlet
         /// </summary>
         [Parameter(Mandatory = true)]
         [ValidateNotNullOrEmpty]
-        public int Seconds { get; set; }
+        public TimeSpan Duration { get; set; }
```

Comment:
We could in fact allow both `Seconds` and `Duration` (and make them mutually exclusive via PowerShell parameter sets). But this is a minor thing, let's leave it as is for now.